README.md

```diff
@@ -188,6 +188,28 @@ it('test1', () => doSomething());
 
 If you need to skip test in all browsers without a comment you can use [mocha `.skip` method](http://mochajs.org/#inclusive-tests) instead of `hermione.skip.in(/.*/);`. The result will be the same.
 
+## Sharable meta info
+On initialization hermione adds two commands to the webdriverio instance:
+* setMeta(key, value)
+* getMeta(key)
+
+These methods allow to store some information between webdriver calls so it can be used in custom commands for example. This meta information will be shown in [allure report](https://github.com/gemini-testing/hermione-allure-reporter).
```

Comment:
надо,  кажется, выделить `for example` запятой или даже в начало его вынести, так сейчас перевод такой:

`это может быть использовано в кастомный командах для примера` (какого примера?)

а тут вроде подразумевается:

`это, например, может быть использовано в кастомных командах`

ну типо я бы так написал

`so, for example, it can be used in custom commands`